.github/workflows/android-ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             "!**/*.md"]
 
   android-build:
-    runs-on: MapLibre_Native_Linux_16_core
+    runs-on: ${{ github.event.pull_request && !github.event.pull_request.draft && 'MapLibre_Native_Linux_16_core' || 'ubuntu-22.04' }}
```

Comment:
This github fake ternary is really hard for me to wrap my head around, but based on the samples I've seen it looks correct. Let's see if it works.

Reply:
Seems to work!

Reply:
Actually, maybe it needs some parentheses.